### PR TITLE
fix gitlab source typo

### PR DESCRIPTION
**Pull request by tigrato**
This PR fixes an issue with Gitlab source name where it incorrectly referenced Gitlab source with every letter capitalized,

### CODE REVIEW

File: docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx [hunk 1]
```diff
@@ -113,14 +113,14 @@ You can also run queries to fetch specific information from the Access Graph, su
 The following query fetches all projects accessible to a user <Var name="user" />:
 
 ```sql
-SELECT * FROM access_path WHERE "identity" = '<Var name="user" />' AND source='GITLAB'
+SELECT * FROM access_path WHERE "identity" = '<Var name="user" />' AND source='Gitlab'
```
Comment (ptgott) [inline docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx:116]: The product is called "GitLab". Is "Gitlab" a hardcoded name within Teleport?

Comment (tigrato) [reply docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx:116]: Yes in access graph

**Review by ptgott — APPROVED** [review]

File: docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx (hunk 1 above)
Comment (ptgott) [reply docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx:116]: 👍 For later, is there a way to alias "Gitlab" as "GitLab" so customers can use the correct capitalization alongside the incorrect one?

**Review by marcoandredinis — APPROVED** [review]

File: docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx (hunk 1 above)
Comment (jakule) [reply docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx:116]: Each integration has different casing, we should probably make that field case insensitive (for backward compatibility) and update our docs with lowercase examples to make it simpler for everyone.

**Pull request merged**